autoload/lsp/hover.vim

```diff
@@ -72,12 +72,14 @@ enddef
 
 # process the 'textDocument/hover' reply from the LSP server
 # Result: Hover | null
-export def HoverReply(lspserver: dict<any>, hoverResult: any): void
+export def HoverReply(silent: bool, lspserver: dict<any>, hoverResult: any): void
```

Comment:
Can you remove the hover related changes from this PR?

Reply:
this PR based on #262

Reply:
removed.